tests/nimony/untyped/tuntypedtemplnocompat.nim

```diff
@@ -0,0 +1,18 @@
+import std / [syncio]
+
+type
+  Inner = object
+    s: string
+  Function = object
+    inner: Inner
+
+template emit(s2: string) {.untyped.} = f().inner.s.add s2
```

Comment:
Nim compiles with with `s: string` btw, I think.

Reply:
https://play.nim-lang.org/#pasty=QrXzWNid, also tested devel, doesn't work, some code probably depends on this too